tools/release/generate_issue_content.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+LAST_TAG="v0.4.17"
+NEW_RELEASE="0.4.18"
+FUTURE_RELEASE="0.5.0"
```

Comment:
You might want to use `read` to ask the user to input value. 

Maybe https://github.com/ipython/ipython/blob/main/tools/release_helper.sh can be of inspiration.